frontend/public/components/search.tsx

```diff
@@ -183,17 +183,26 @@ const SearchPage_: React.FC<SearchProps> = (props) => {
     if (!model) {
       return '';
     }
-    const { labelPlural, apiVersion, apiGroup } = model;
+    const { labelPlural, labelPluralKey, apiVersion, apiGroup } = model;
     return (
       <span className="co-search-group__accordion-label">
-        {labelPlural}{' '}
+        {labelPluralKey ? t(labelPluralKey) : labelPlural}{' '}
         <span className="text-muted show small">
           {apiGroup || 'core'}/{apiVersion}
         </span>
       </span>
     );
   };
 
+  const getChipText = (item: string) => {
+    const model = modelFor(item);
```

Comment:
Don't we already have the model? We shouldn't need to find it again.

Reply:
We don't store it; it's in a helper function. I can try refactoring tomorrow.

Reply:
I looked into refactoring, but I think it would a lot of complexity to this component. This component pretty much just deals with a reference rather than the kind. Like getToggleText, you need to convert the reference into the model to get the correct value.